tests/terraform/createcluster/createcluster_test.go

```diff
@@ -102,18 +110,22 @@ var _ = Describe("Test:", func() {
 			Expect(err).NotTo(HaveOccurred(), "Cluster IP manifest not deployed")
 
 			Eventually(func(g Gomega) {
-				cmd := "kubectl get pods -o=name -l k8s-app=nginx-app-clusterip --field-selector=status.phase=Running --kubeconfig=" + KubeConfigFile
+				cmd := "kubectl get pods -o=name -l k8s-app=nginx-app-clusterip" +
```

Comment:
I left a similar comment in the RKE2 PR, but I will repeat it here

There multiple times in the test where your basically asking for the same thing: "Is this pod running?", it might be easier to just consolidate all into a common function

```
func IsAppRunning(namespace, appName string) (string, error) {
    cmd := "kubectl get pods -n" + namespace + " -o=name -l k8s-app=" + appName + " --field-selector=status.phase=Running --kubeconfig=" + KubeConfigFile
    return RunCommand(cmd)
}
```

Reply:
this would need another refactor i believe, since its not my code. it was already there and i passed from this, since my pr was not about fixing all possible refactors ...just critical one and related to the stuff i was changing. 
this line i just break it for better readability